testrail-connector/src/main/java/com/nullin/testrail/TestRailReporter.java

```diff
@@ -206,15 +207,15 @@ public void reportResult(String automationId, Map<String, Object> properties) {
 				return; // nothing more to do
 			}
 
-			StringBuilder comment = buildComments(moreInfo, screenshotUrl, resultStatus, throwable);
+			StringBuilder comment = buildComments(moreInfo, screenshotUrl, resultStatus,primaryDevice, throwable);
 
 			// add the result
 			Map<String, Object> body = new HashMap<String, Object>();
 			body.put("status_id", getStatus(resultStatus));
 			body.put("comment",
 					new String(comment.toString().getBytes(StandardCharsets.ISO_8859_1), StandardCharsets.UTF_8));
 			body.put("elapsed", elapsed);
-
+			body.put("custom_test_device", 999);//comment added
```

Comment:
Since this is hard-coded field name, let's move it as constant where it's easy to find. Or makes it as another argument with default value.